client/src/components/landing/LandingNavigationBar.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <div class="mj-landing-navbar">
-    <application-title
-      :is-nav-bar="true"
-      :with-icon="true" />
+    <a onclick="window.location.reload()">
+      <application-title
+        :in-nav-bar="true"
+        :with-icon="true" />
+    </a>
     <div class="mj-landing-navbar-buttons">
```

Comment:
I would just remove this code. Just add "cursor: pointer". `a` with event `onclick` is worse. `a` should handle `href` param to be treated as a link. Or use vue's `@click`, just don't use plain js when we use a framework.